### DISCUSSION TITLE
how to force elements to be in the same line without breaking the line when window size changes smaller?

### DISCUSSION THREAD

**Discussion by yanghw116** [zauberzeug/nicegui#2478] [Q&A] [answered]
### Question

hi everyone, I'm new to nicegui.

example code here
```py
from nicegui import ui

columns = [
    {'name': 'name', 'label': 'Name', 'field': 'name', 'required': True, 'align': 'left'},
    {'name': 'age', 'label': 'Age', 'field': 'age', 'sortable': True},
]
rows = [
    {'name': 'Alice', 'age': 18},
    {'name': 'Bob', 'age': 21},
    {'name': 'Carol'},
]

with ui.row().classes('w-full justify-between'):
    ui.table(columns=columns, rows=rows, row_key='name')
    ui.table(columns=columns, rows=rows, row_key='name')
    ui.table(columns=columns, rows=rows, row_key='name')
    ui.table(columns=columns, rows=rows, row_key='name')
    ui.table(columns=columns, rows=rows, row_key='name')
    ui.table(columns=columns, rows=rows, row_key='name')

ui.run()
```

I create many elements in a row. but when the window size becomes smaller, they are rearranged to multiple rows. what I expect is that they stay in one row and a scrollbar is placed in the footer to indicate the range. can anyone helps me? thanks in advance.

**Comment by yanghw116** [comment]
seems `w-max` solves the problem. but `w-max` disables `justify-between`. can they be centered in the window when the window width is larger than the total width of elements in a row?

**Comment by yanghw116** [comment, marked as answer]
`w-max justify-center mx-auto` solves the problem.

**Comment by falkoschindler** [comment]
It sounds like you're looking for `ui.row(wrap=False)`.

**Reply by yanghw116** [reply]
Definitely, `ui.row(wrap=False).classes('w-full justify-center')` did the same thing.
